src/pages/VideoCall/index.js

```diff
@@ -94,17 +83,10 @@ const VideoCallFrame = () => {
         eventActions()
 
         daily.on('joined-meeting', eventActions)
-
-        daily.on('participant-updated', eventActions)
-
-        daily.on('participant-left', (event) => { 
-
-            if (event.participant.local === false && Object.keys(participants).length < 2) {
-                setParticipantStatus(t('participant-left'))
-            } else {
-                setParticipantStatus('')
-            }
-        })
+            .on('participant-updated', eventActions)
+            .on('participant-left', (event) => { 
+                setParticipantStatus(!event.participant.local && Object.keys(participants).length < 2 ? t('participant-left') : '')
```

Comment:
`participantsLength` is define above use it instead ` Object.keys(participants).length `

Reply:
`participantsLength` is defined in the `eventActions` method. However, `eventActions` is not called when `participant-left` event occurs, hence why `Object.keys(participants).length` is used. 

To avoid this and to be able to use `participantsLength` everywhere, I tried to:

- define `participantsLength` outside of the eventActions method
- put participantsLength in the state (as such: `const [ participantsLength, setParticipantsLength ] = useState(0) `)

But then, the participant status message is not displayed properly, so I kept the code as is.

Reply:
Ok